mne/decoding/transformer.py

```diff
@@ -147,6 +147,8 @@ def inverse_transform(self, epochs_data, y=None):
         return X
 
 
+@deprecated("Class 'ConcatenateChannels' has been renamed to "
+            "'EpochsVectorizer' and will be removed in release 0.11.")
 class ConcatenateChannels(TransformerMixin):
```

Comment:
can't you just do:

```
@deprecated("Class 'ConcatenateChannels' has been renamed to  ...")
class ConcatenateChannels(EpochsVectorizer):
     pass
```

in order to remove the old code of ConcatenateChannels?